p2p_wallet/Scenes/Main/SendToken/Subscenes/ChooseRecipientAndNetwork/SelectAddress/SendToken.ChooseRecipientAndNetwork.SelectAddress.ViewModel.swift

```diff
@@ -130,48 +136,51 @@ extension SendToken.ChooseRecipientAndNetwork.SelectAddress
         chooseRecipientAndNetworkViewModel.feeInfoDriver
     }
 
+    var warning: Driver<String> {
+        Driver<Bool>
+            .just(amount < minSolForSending)
```

Comment:
warningDriver is not needed in this context, just use normal var warning: String?
Optional is suitable, because warning can be missing

Reply:
we're going to have dirty init because of this, I think, that Driver will be better. And it will be easy refactored later, what do you think?

Reply:
we don't have amount any, because of this I passed amount via init, but if we have single wallet and walletDriver, it's something unusual and it seems incorrect

Reply:
but for passing only is valid amount, previous viewModel must contain logic for formatting this parameter, it's not correct, this logic must be contained in this viewModel